Android/Fundamentals/intro-to-android/what-is-android.md

```diff
@@ -0,0 +1,49 @@
+# What is Android?
+author: tlawson
+
+levels:
+
+  - beginner
+
+type: normal
+
+category: must-know
+
+links:
+
+  - '[What is Android?](https://developer.android.com/guide)'
+
+---
+## Content
+
+Android is a software package for mobile devices including computers, smartphones, and tablets.
```

Comment:
might be more appropriate to categorize Android as an 'operating system' as opposed to a 'software package'